### PR TITLE
15 fix improving get metadata

### PR DESCRIPTION
**Pull request by BorjaEst**
temp fast fix for metadata key conflict between ai4 and importlib (pyproject.toml)

### CODE REVIEW

**Comment by BorjaEst** [comment]
Solves #9 and #14

File: {{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/config.py [hunk 1]
```diff
@@ -17,28 +17,19 @@
...
-_AUTHORS = [] if _AUTHORS == [""] else _AUTHORS
-_AUTHORS += PROJECT_METADATA["Author-emails"].keys()
-PROJECT_METADATA["Authors"] = sorted(_AUTHORS)
-
 # Get ai4-metadata.yaml metadata
-AI4_METADATA_DIR = os.getenv(
-    f"{API_NAME.capitalize()}_AI4_METADATA_DIR",
-    default=f"{os.getcwd()}/../{API_NAME}",
-)
+CWD = os.getcwd()
```
Comment (BorjaEst) [inline {{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/config.py:21]: This checks "ai4-metadata.yml" in:
 - $cwd/name_of_module
 - $cwd/../name_of_module which should be eq to run deepaas in any of the modules folders

File: {{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/config.py [hunk 2]
```diff
@@ -17,28 +17,19 @@
...
+    if "ai4-metadata.yml" in os.listdir(f"{CWD}/{API_NAME}"):
+        AI4_METADATA_DIR = f"{CWD}/{API_NAME}"
+    elif "ai4-metadata.yml" in os.listdir(f"{CWD}/../{API_NAME}"):
+        AI4_METADATA_DIR = f"{CWD}/../{API_NAME}"
+
+# Open ai4-metadata.yml
 with open(f"{AI4_METADATA_DIR}/ai4-metadata.yml", "r", encoding="utf-8") as stream:
     AI4_METADATA = yaml.safe_load(stream)
 
-# Merge metadata from pyproject.toml and ai4-metadata.yml
```
Comment (BorjaEst) [inline {{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/config.py:39]: Remove merge due to key conflicts between 2 metadatata packages

File: {{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/api.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def get_metadata():
     """
     try:  # Call your AI model metadata() method
         logger.info("Collecting metadata from: %s", config.API_NAME)
```
Comment (BorjaEst) [inline {{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/api.py:52]: Return only ai4-metadata

File: {{ cookiecutter.__repo_name }}/ai4-metadata.yml [hunk 4]
```diff
@@ -6,8 +6,8 @@ description: |-
   {{ cookiecutter.description.replace('\'', '') }}
 # doi: http://add-some-DOI-url.com
 dates:
```
Comment (BorjaEst) [inline {{ cookiecutter.__repo_name }}/ai4-metadata.yml:8]: fixes from #14

**Comment by IgnacioHeredia** [comment]
Hi @BorjaEst , I'm fine with the changes (ie. return only ai4-metadata, not toml metadata).
Butt maybe cross-check with @vykozlov before merging 😉

**Comment by BorjaEst** [comment]
I will before merge, thanks! 😀

**Pull request merged**